scripts/ingests/Gaia/ingest_gaia.py

```diff
@@ -0,0 +1,362 @@
+    gaia_query_string = "SELECT *,upload_table.db_names FROM gaiaedr3.gaia_source " \
+                             "INNER JOIN tap_upload.upload_table ON " \
+                        "gaiaedr3.gaia_source.source_id = tap_upload.upload_table.dr3_source_id  "
+    job_gaia_query = Gaia.launch_job(gaia_query_string, upload_resource=input_table,
+                                     upload_table_name="upload_table", verbose=VERBOSE)
+
+    gaia_data = job_gaia_query.get_results()
+
+    print('Gaia eDR3 query complete')
+
+    return gaia_data
+
+
+# query eDR3
+# gaia_edr3_data = query_gaiaedr3(gaiadr3_names)
+edr3_data_file_string = 'scripts/ingests/Gaia/gaia_edr3_data_'+DATE_SUFFIX+'.xml'
+# gaia_edr3_data.write(edr3_data_file_string, format='votable')
+# read results from saved table
+gaia_edr3_data = Table.read(edr3_data_file_string, format='votable')
+
+
+# add Gaia telescope, instrument, and Gaia filters
+def update_ref_tables():
+    add_publication(db, doi='10.1051/0004-6361/201629272', name='Gaia')
+    db.Publications.delete().where(db.Publications.c.name == 'GaiaDR2').execute()
+    db.Publications.update().where(db.Publications.c.name == 'Gaia18').values(name='GaiaDR2').execute()
+    add_publication(db, doi='10.1051/0004-6361/202039657', name='GaiaEDR3')
+
+    gaia_instrument = [{'name': 'Gaia', 'reference': 'Gaia'}]
+    gaia_telescope = gaia_instrument
+    db.Instruments.insert().execute(gaia_instrument)
+    db.Telescopes.insert().execute(gaia_telescope)
+
+    gaia_filters = [{'band': 'GAIA2.G',
+                     'instrument': 'Gaia',
+                     'telescope': 'Gaia',
+                     'effective_wavelength': 6230.,
+                     'width': 4183.},
+                    {'band': 'GAIA2.Grp',
+                     'instrument': 'Gaia',
+                     'telescope': 'Gaia',
+                     'effective_wavelength': 7730.,
+                     'width': 2757.},
+                    {'band': 'GAIA3.G',
+                     'instrument': 'Gaia',
+                     'telescope': 'Gaia',
+                     'effective_wavelength': 5822.,
+                     'width': 4053.},
+                    {'band': 'GAIA3.Grp',
+                     'instrument': 'Gaia',
+                     'telescope': 'Gaia',
+                     'effective_wavelength': 7620.,
+                     'width': 2924.}
+                    ]
+
+    gaia_dr3filters = [{'band': 'GAIA3.G',
+                     'instrument': 'Gaia',
+                     'telescope': 'Gaia',
+                     'effective_wavelength': 5822.,
+                     'width': 4053.},
+                    {'band': 'GAIA3.Grp',
+                     'instrument': 'Gaia',
+                     'telescope': 'Gaia',
+                     'effective_wavelength': 7620.,
+                     'width': 2924.}
+                    ]
+
+    db.PhotometryFilters.insert().execute(gaia_filters)
+    db.PhotometryFilters.insert().execute(gaia_dr3filters)
```

Comment:
An alternative to this would have been to modify https://github.com/SIMPLE-AstroDB/SIMPLE-db/blob/main/scripts/ingests/add_photometry_filters.py and add a few extra lines of the form:
```
{'telescope': 'Gaia', 'instrument': 'Gaia', 'filter': 'G'},
{'telescope': 'Gaia', 'instrument': 'Gaia', 'filter': 'Grp'},
```
since that scripts automatically adds missing Telescopes, Instruments, PhotometryFilters, if they don't exist, and it queries the SVO to get the wavelength/width information.
The only thing I see that would need verification is that it expects to construct the band name from the instrument+filter so the `fetch_svo` function would need some minor edits.

Reply:
Huh, ok, I will keep this is mind for future ingests.